components/recap/thankyou.js

```diff
@@ -20,13 +20,16 @@ import {
   Textarea
 } from 'theme-ui'
 
+import usePrefersMotion from './use-prefers-motion'
+import useHasMounted from './use-has-mounted'
+
 const Collab = ({ img, alt, url }) => (
-  <Link href={url} target="_blank" sx={{ display: 'block', mx: 3, mb: [3, 4] }}>
+  <Link href={url} rel="noopener" target="_blank" sx={{ display: 'block', mx: 3, mb: [3, 4] }}>
     <Image src={img} alt={alt} sx={{ maxWidth: [112,224,224], maxHeight: [42,84,84] }} />
   </Link>
 )
 
-export default () => (
+const SheetOLd = () => (
```

Comment:
Is this component being used?

Reply:
nope, I'm leaving it there for now in case I need to reference something from it.. I'll put it into a file on my computer